setup.cfg

```diff
@@ -0,0 +1,29 @@
+[pytest]
+norecursedirs =
+    .git
+    dist
+    build
+    venv
+    .env
+testpaths =
+    tests
+python_files = *Test.py
+python_classes = *Test
+python_functions = *_test
+timeout = 35
```

Comment:
You can reduce the timeout in coala-html
The 35 was mainly for integration tests that dont exist here.

You could just do `5`